localstack/services/lambda_/invocation/lambda_models.py

```diff
@@ -64,8 +64,9 @@
     "go1.x": "go:1",
     "provided": "provided:alami",
     "provided.al2": "provided:al2",
+    "java21": "java:21",
 }
-SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17]
+SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17, Runtime.java21]
```

Comment:
great catch 👍 that reminded me about SnapStart when re-organizing the Lambda runtimes.